ssl/ssl_ciph.c

```diff
@@ -498,12 +499,12 @@ static void do_load_builtin_compressions(void)
         }
     }
     CRYPTO_mem_ctrl(CRYPTO_MEM_CHECK_ENABLE);
+    return 1;
```

Comment:
I somehow missed the `ssl_comp_methods != NULL` check.

Anyway, in general I think it's up to the caller to decide what to do in case of an error, not the callee.  In this case the caller (3 lines lower) just ignores it, so I guess we can just ignore it here too.